npm/src/lib/node.js

```diff
@@ -30,6 +30,13 @@ export const fs = {
       });
     });
   },
+  readFile: function(f: string, opts: Object): Promise<Buffer> {
```

Comment:
Mostly because Flow doesn't have a libdef for things like promisifyAll (which I think could be a little tricky to write). This way the API stays well typed.